youtube_dl/extractor/americastestkitchen.py

```diff
@@ -0,0 +1,53 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+
+
+class AmericasTestKitchenIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?americastestkitchen\.com/episode/(?P<id>\d{3}(-\w+)+)'
```

Comment:
`id` is of arbitrary length.

Reply:
Correct me if I am wrong, but this _VALID_URL regex is used for determining whether a URL matches this extractor. If that is correct, then having an arbitrary number of words separated by a dash is correct for this site.

Reply:
`id` is **not necessarily** 3 digits.

Reply:
You are correct on the 3 digits. I have fixed that, just not pushed it. The id noted here is between 1 & 3 digits followed by a dash, and then an unknown number of words separated by a dash. 

So I'm not clear on what is meant by 'irrelevant words'. Could I get some clarification around that?

Reply:
`id` must not include anything apart from digits.

Reply:
Oh, ok. I'll fix that. Thanks.

Reply:
Ok. I've updated this. It should match what we discussed here.